docs/pywbemclicmdshelp.rst

```diff
@@ -191,9 +191,9 @@ The following defines the help output for the `pywbemcli class associators --hel
       -c, --resultclass <class name>  Filter by the result class name provided.
       -r, --role <role name>          Filter by the role name provided.
       -R, --resultrole <role name>    Filter by the role name provided.
-      --includequalifiers / --no_includequalifiers
-                                      Include qualifiers in the result. Default is
-                                      to include qualifiers
+      --no-qualifiers                 Do not include qualifiers in the
+                                      response.The default behavior is to
+                                      includequalifiers in the returned class.
```

Comment:
The description talks about both "response" and "returned class". The response is not always a class, so this inconsistency is confusing. Can that be made consistent?

Reply:
This gets messy.  We separated the class level and instance level requests in the cli despite the fact that they call the same entity. Further, we concluded that for the class level, the default is to return qualifiers whereas the for the instance level it is nto not include qualifiers. Thus, in the class help the  flag is --no-qualifiers and there is no short form under the assumption that it is seldom used.

For instances, the world is inverted. Normally qualifiers are of no interest so the flag is -q/--includqualifiers and the default is to not return qualifiers.  Actually the return is always a class for the class level command and an instance for the instance level response. Note however, that the code does not actually enforce the limit for references and associatiors (i.e. we do not test for instance id)

Changed the word response to result. Changed the word class to objects. Note that this single option is defined once and applied to several different subcommands in the _cmd_class group.

Reply:
Thanks for clarifying that. See my new comments on that.